devTools/calibrationUtils.py

```diff
@@ -87,10 +88,106 @@ class Observation:
 
 @dataclass
 class CameraCalibration:
-    resolution: Resolution
+    resolution: Size
     cameraIntrinsics: JsonMatOfDoubles
     distCoeffs: JsonMatOfDoubles
     observations: list[Observation]
+    calobjectWarp: list[float]
+    calobjectSize: Size
+    calobjectSpacing: float
+
+
+import mrcal
+from wpimath.geometry import Quaternion as _Quat
+
+
+def __convert_cal_to_mrcal_cameramodel(
+    cal: CameraCalibration,
+) -> Union[mrcal.cameramodel, None]:
```

Comment:
If we are using Python 3.11, just use `mrcal.cameramodel | None`, you don't need to use Union